### ISSUE TITLE
[Supporting the platform - Edit Panel]: Controls present under "Time series graph/Edit panel" blade are not visible after setting the view port at 320*256.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66582]
## User Experience: 
Low vision users who depend on zooming the page for better readability will be impacted as they will not be able to see the complete content present in the page if the controls are not visible or truncated which will result in lack of understandability and will fail to access the controls. 

## Pre-requisite:  
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

## Repro Steps:
Open #Url: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the Feature Showcase pane heading and select it. Navigate the expanded Dropdown and select the "Edit" menu item. Navigate the page "Edit Panel" page which appears. Select "1Time Series graphs" Navigate to the "Override" button and select it.
Navigate the options present under the "Override" pane. Navigate to the "Add Field Override" button and select it. Verify the issue.

## Actual Result: 
Controls such as "Table View", "Fill/Actual", "Query", "Transform" and "Alert" ...etc. present under "Time series graph/Edit panel" blade are not visible after setting the view port at 320*256.

## Expected Result:
 Controls such as "Table View", "Fill/Actual", "Query", "Transform" and "Alert"...etc. present under "Time series graph/Edit panel" blade should be clearly visible after setting the view port at 320*256. 
Note: The same issue exists under 'Grafana play home/Edit panel' screen. (Home>select feature showcases>edit) Same issue appears for Edit ->All->Value mapping. Same issue appears for Edit ->All->Thresholds. Same issue appears for edit->all->override
 MAS 1.4.10 – Reflow
